test/JDBC/expected/temp_catalog_inval-vu-verify.out

```diff
@@ -0,0 +1,59 @@
+
+-- Catalog entries for temp tables are not visible from places like `pg_class` or `information_schema.tables`
+-- So the best way to check whether the cache entries have been updated locally properly is to just ensure that
+-- it's visible in typical use. 
+-- pg_class, pg_type, pg_attribute
+CREATE TABLE #t1(a int)
+go
+
+insert into #t1 values (1)
+go
+~~ROW COUNT: 1~~
+
+
+select * from #t1
+go
+~~START~~
+int
+1
+~~END~~
+
+
+drop table #t1
+go
+
+
+-- constraint, depend, index, sequence, attrdefault
+create table #t2(a int identity primary key, b char(15) default 'column default')
```

Comment:
this is not clear. Ideally we want to test use case where an existing entry is invalidated. create table will add new entries. How do we validate that existing entries are invalidated?
drop table is one use case where we try to do select after drop. Can we find some alter commands which invalidate existing cat cache entries and later do a select

Reply:
ack. added some alter table and drop calls for the above.